utils/build-presets.ini

```diff
@@ -890,6 +889,8 @@ installable-package=%(installable_package)s
 # in Linux CI bots
 relocate-xdg-cache-home-under-build-subdir
 
+# Linux only support 'hosttools' with early-swiftsyntax.
+bootstrapping=hosttools
```

Comment:
We're going to need to add this in quite a few places. Maybe we should override and set to hosttools when early swiftsyntax is enabled, since that's the only thing that works? Bit weird but...

Reply:
Updated to force setting `set(BOOTSTRAPPING_MODE "HOSTTOOLS")` when `SWIFT_SWIFT_PARSER` is enabled.